ui/widget.js

```diff
@@ -366,17 +368,39 @@ $.Widget.prototype = {
 
 		return this;
 	},
+	_updateClassesOption: function( value ) {
+		var classKey, elements, currentElements;
+		for ( classKey in value ) {
+			currentElements = this.classObject[ classKey ];
+			if ( value[ classKey ] === this.options.classes[ classKey ] ||
+					!currentElements ||
+					!currentElements.length ) {
+				continue;
+			}
+			elements = $( currentElements.get() );
+			this._removeClass( currentElements, classKey );
+			elements.addClass( this._classes({
```

Comment:
Yes i'v answered both you and @scottgonzalez and since you don't remember and are asking again that means i clearly need a comment here to explain.

`_addClass\removeClass\_toggleClass` all omit the `classes` property of the the object being passed `_classes` making it use `this.options.classes` Here we don't want to use the current `this.options.classes` we want to use what will be the new value which was passed to `_setOption`